hack/verify-readme.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -eu
+
+rootdir="$(echo $(cd $(dirname $0) && pwd))/.."
+
+cd "$rootdir"
+before=$(cat README.md)
+make update-readme
+after=$(cat README.md)
+
+if [[ "$before" != "$after" ]]; then
+  echo "Error: Running update-readme makde a diffrence in README.md."
+  echo "Maybe you forgot run 'make update-readme'."
```

Comment:
Since the other suggestion will make `verify-readme.sh` not modify README.md, I suggest the following change:

```suggestion
  echo "Run 'make update-readme'" >&2
```

Reply:
fixed and rebased 👍 4f321af